mx.truffleruby/mx_truffleruby.py

```diff
@@ -68,9 +68,14 @@ def ruby_testdownstream(args):
     mx.command_function('build')([])
     mx.run(['ruby', 'tool/jt.rb', 'test', 'fast'])
```

Comment:
We could also make it part of the command above and accept arguments like `specs`, `hello`, etc
@nirvdrum What do you think?